src/garrysmod/gamemodes/zombiesurvival/gamemode/zombieclasses/zombie_gore_blaster.lua

```diff
@@ -13,6 +13,7 @@ CLASS.Unlocked = true
 CLASS.Health = 220
 CLASS.Speed = 180
 CLASS.Revives = false
+CLASS.Model = Model("models/player/zombie_classic_hbfix.mdl")
```

Comment:
Is there a reason we're not also setting `CLASS.UsePlayerModel = false`?

Reply:
`UsePlayerModel` is ignored if `Model` is defined. I could set it to false for clarity, but it wont change the behaviour

Reply:
Yeah, that makes sense. I see two paths. We could set it to `false` so that we can depend on `CLASS.UsePlayerModel` being available for certain logic. At the same time, we could also just check `CLASS.Model ~= nil` to (maybe) get a similar result. I say maybe, because there may be cases where `Class.Model == nil` but it doesn't mean we're using a player model.

Up to you.